config/datastores/formatters.yaml

```diff
@@ -10,11 +10,6 @@ PhotoCalib: lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter
 TransmissionCurve: lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter
 Camera: lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter
 Detector: lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter
-ImageF: lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter
-MaskX: lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter
-Exposure: lsst.obs.base.fitsExposureFormatter.FitsExposureFormatter
-ExposureF: lsst.obs.base.fitsExposureFormatter.FitsExposureFormatter
-ExposureI: lsst.obs.base.fitsExposureFormatter.FitsExposureFormatter
```

Comment:
These get picked up because of StorageClass inheritance, I assume?

Reply:
They are duplicate entries in the YAML that were just confusing things. Copy and paste error I think. They weren't doing anything useful.